frontend/src/communications/mockForUserApi.js

```diff
@@ -1,10 +1,10 @@
-import api from './apiInstance'
-import MockAdapter from 'axios-mock-adapter'
+// import api from './apiInstance'
+// import MockAdapter from 'axios-mock-adapter'
 
-const mock = new MockAdapter(api)
+// const mock = new MockAdapter(api)
```

Comment:
Töröljük teljesen vagy ha kell még később, akkor csak ne importáljuk.

Reply:
Töröltem, mert hiába nem importáljuk az API-knál, mégis behúzza a mock hívásokat és nem érjük el a BE végpontokat.